test/integration/client-side-encryption/client_side_encryption.prose.23.range_encryption_defaults.test.ts

```diff
@@ -0,0 +1,144 @@
+import { type Binary, EJSON, Int32, Long } from 'bson';
```

Comment:
We should import BSON from the driver here, to match our expectation of the user's usage

Reply:
That's not something we enforce in our tests, just searching `from 'bson'` yields a bunch of results.  I'm also not convinced of its value tbh.  But regardless, unless you'd like me to fix all occurrences of importing from bson directly in this PR, I'm going to leave this as-is.

Reply:
Our tests should match the way users use the API, for now there's no bug and I am sure we have some mistakes around our tests where we don't catch this. But we ask our users to use the BSON exported from the driver for correctness reasons the tests should match otherwise they don't reproduce the expected use case. 

I think it is acceptable to just address the new code added in the PR, but feel free to fix more than asked if you feel so inclined.

Reply:
Done.